app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (<h1>Hello World!</h1>);
```

Comment:
- [ ] It would be better if our App component uses routes from `react-router` and also a greeting component imported from the components directory :100: